CMakeLists.txt

```diff
@@ -36,8 +36,8 @@ if (BUILD_DEPS)
 
     add_subdirectory(crt/aws-c-common)
     if (UNIX AND NOT APPLE)
-        list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/crt/s2n/cmake/modules")
-        find_package(LibCrypto REQUIRED)
+        message(STATUS "Pre aws-lc CMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}")
+        include(crt/aws-c-cal/cmake/modules/aws-lc.cmake)
```

Comment:
I have a PR that will speed up aws-lc builds. Steps required:
1) Sync aws-lc submodule to latest on main branch
2) This PR https://github.com/awslabs/aws-c-cal/pull/74 needs to be merged, and a new aws-c-cal release needs to be cut.
3) Sync aws-c-cal submodule

Reply:
PR is merged. sync aws-c-cal to v0.5.2

Reply:
Need to reject this for now. When I update, I run into encryption failures to be addressed separately.